system-int-test/test/cmr/system_int_test/bootstrap/bulk_index/subscriptions_test.clj

```diff
@@ -48,12 +48,16 @@
      (let [coll1 (d/ingest-umm-spec-collection "PROV1" (data-umm-c/collection {:EntryTitle "E1"
                                                                                :ShortName "S1"
                                                                                :Version "V1"}))
-           sub1 (subscription/ingest-subscription-with-attrs {:provider-id "PROV1" :CollectionConceptId (:concept-id coll1)}
+           sub1 (subscription/ingest-subscription-with-attrs {:provider-id "PROV1"
+                                                              :Query "limit=1"
```

Comment:
nit: I wish the `Query` could be more realistic, nobody would create a subscription with a query of `limit=1`, something like `short_name=x` would be better.

Reply:
I have done a pass on all the tests to use realistic queries.